packages/northstar-wrapper/package.json

```diff
@@ -3,36 +3,43 @@
   "version": "1.5.1-beta.4",
   "private": "true",
   "description": "Package to wrap the Fluent NorthStar UI library, for short-term React 18 compatibility",
+  "module": "dist/dist-esm/index.js",
+  "main": "dist/dist-cjs/index.js",
+  "types": "dist/northstar-wrapper.d.ts",
   "scripts": {
-    "build": "tsc",
+    "build": "rushx build:esm && rushx build:cjs",
+    "build:esm": "tsc && webpack -c ./webpack.esm.config.js && rushx api-extractor",
```

Comment:
If that is esm, will webpack bundle them into one file or several files like all the other packages?

Reply:
Still one file, but a file that has treeshaking support (treeshaking can still strip out unused exports in the same file)

Reply:
Lets open a work item to validate that later including find a solution for treeshaking localization part